src/Command/SetupCommand.php

```diff
@@ -148,40 +157,39 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $gatewayConfig
         );
 
-        $output->writeln('Successfully created microservice settings');
+        $io->success('Successfully created microservice settings');
 
         $this->release();
     }
 
     private function generateConfigFile(
         string $gatewayDirectory,
-        string $httpPort,
-        string $httpsPort
+        string $httpPort = null,
+        string $httpsPort = null
     ): string {
-        $config = <<<EOT
-version: '2'
-
-services:
-  nginx:
-    image: prooph/nginx:www
-    volumes:
-      - ./$gatewayDirectory:/etc/nginx/sites-enabled:ro
-    labels:
-      prooph-gateway-directory: ./$gatewayDirectory
-EOT;
-
-        if ($httpPort !== '80' || $httpsPort !== '443') {
-            $config .= "\n    ports:\n";
-        }
-
-        if ($httpPort !== '80') {
-            $config .= "      - $httpPort:80\n";
+        $config = [
+            'version' => '2',
+            'services' => [
```

Comment:
Not sure if it was already discussed, but we should use version 3 (requires Docker 1.13 and Docker Composer 1.10) but then we can deploy the stuff to a Swarm Cluster with `docker stack deploy --compose-file=docker-compose.yml my_stack`.